config_templates/gretel/navigator_ft/time-series.yml

```diff
@@ -0,0 +1,11 @@
+schema_version: "1.0"
+name: "time-series"
```

Comment:
A couple thoughts:

- Maybe we should include `navigator-ft` in the name? This name will appear in the Console. Actually, I guess the full name would be `navigator_ft/tabular`? That works for the high-level SDK, but not sure about the Console. 

- I wonder if we should be more general than `time-series`. e.g., `sequential`? Ordering is also optional, but maybe we don't have to get into that.

Reply:
- The pattern I see in other paths is to not include the top level name, so I think the full name will end up becoming `navigator_ft/tabular`
- time-series probably conveys the sequential nature? Also, isn't the goal of these blueprints to provide more specific examples?

Reply:
The goal is to provide a simple default config that users can build upon. IMO "time series" is more specific than we want to be here. e.g., the ordered column doesn't necessarily have to be time (actually there doesn't even need to be an ordering at all, which is another consideration). I also think people put "time series"  and "event data" in separate bins in their mind, and "event data" is more aligned with what this model will be good at (at least initially). My thinking that "sequential" is general enough to encompass all ordered data.